### PR TITLE
[HACKADOC] Azure Automation Getting Started Page

### PR DESCRIPTION
**Pull request by alstab**
Optimizing the "getting started"/tutorial page for Azure Automation

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@alstab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by mollybostic** [comment]
@alstab - "Next steps" is a standard heading that we use across the site to refer people to related articles. Do you mind changing your "Resources" heading back to "Next steps"?

File: articles/automation-create-runbook-from-samples.md [hunk 1]
```diff
@@ -13,50 +13,23 @@
...
 
 ## Tutorial
-This tutorial walks you through the steps to create an automation account and then import a sample "Hello World" runbook into Azure Automation, execute the runbook, and then view its output.
+This tutorial walks you through creating an automation account, importing a sample "Hello World" runbook into Azure Automation, executing that runbook, and then viewing its output.
 
-To complete this tutorial, you need an Azure account. If you have not already done so, you can <a href="/en-us/pricing/member-offers/msdn-benefits-details/?WT.mc_id=A85619ABF" target="_blank">activate your MSDN subscriber benefits</a> or <a href="/en-us/pricing/free-trial/?WT.mc_id=A85619ABF" target="_blank">sign up for a free trial</a>.
+To complete this tutorial, you will need an Azure subscription. If you don't have one yet, you can <a href="/en-us/pricing/member-offers/msdn-benefits-details/?WT.mc_id=A85619ABF" target="_blank">activate your MSDN subscriber benefits</a> or <a href="/en-us/pricing/free-trial/?WT.mc_id=A85619ABF" target="_blank">sign up for a free trial</a>.
```
Comment (mollybostic) [inline articles/automation-create-runbook-from-samples.md]: Please remove the campaign ID from your link (WT.mc_id=A85619ABF).

File: articles/automation-create-runbook-from-samples.md [hunk 2]
```diff
@@ -13,50 +13,23 @@
 	ms.tgt_pltfrm="na" 
 	ms.devlang="na" 
 	ms.topic="article"
```
Comment (mollybostic) [inline articles/automation-create-runbook-from-samples.md]: you can set the ms.topic value to hero-article

File: articles/automation-create-runbook-from-samples.md (hunk 1 above)
Comment (alstab) [inline articles/automation-create-runbook-from-samples.md]: We have both a "Next Steps" section and a "Resources" section; I actually didn't rename either of them, just moved them. The Next Steps section is more related to what to do after the tutorial, while Resources is more for forums, blogs, etc. Do we need to change this?

**Comment by mollybostic** [comment]
Having both Next Steps and Resources is fine, thanks! can you also take a look at the inline comments that I added?

**Comment by alstab** [comment]
I think I just addressed these -- you mean the hero-article and campaign ID notes?

**Comment by mollybostic** [comment]
changes look great, thanks!

**Pull request merged**